### PR TITLE
Google analysis 4 implemented

### LINKED ISSUE #712: Update Google Analytics Integration to GA4

### PR DESCRIPTION
**Pull request by ashishpachauriap**
close #712

### CODE REVIEW

**Review by nikhilmandeeps — APPROVED** [review]

File: Portals/_default/Skins/Vanjaro/Base.ascx.cs [hunk 1]
```diff
@@ -252,26 +264,6 @@ private void HandleUXM()
...
-                    {
-                        WebForms.RegisterClientScriptBlock(Page, "GTagManager", "<script type='text/javascript' async src='https://www.googletagmanager.com/gtag/js?id=" + Analytics.MeasurementID + "'></script>", false);
-                        //WebForms.RegisterStartupScript(Page, "GTagManagerScript", "window.dataLayer = window.dataLayer || []; function gtag() { dataLayer.push(arguments); } gtag('js', new Date()); gtag('config', '" + Analytics.MeasurementID + "', { 'send_page_view': false }); " + PostEvent, true);
-                        string script = @"$(document).ready(function () {window.dataLayer = window.dataLayer || []; function gtag() { dataLayer.push(arguments); } gtag('js', new Date()); gtag('config', '" + Analytics.MeasurementID + "', { 'send_page_view': false }); " + PostEvent + "});";
-                        WebForms.RegisterStartupScript(Page, "AnalyticsPostEventScript", script, true);
-                    }
-                }
-            }
-        }
-
```
Comment (meetmandeep) [inline Portals/_default/Skins/Vanjaro/Base.ascx.cs]: @ashishpachauriap Why was this removed? @hshimanshusharma Please review this code

**Review by meetmandeep — DISMISSED** [review]

File: Portals/_default/Skins/Vanjaro/Base.ascx.cs (hunk 1 above)
Comment (ashishpachauriap) [reply Portals/_default/Skins/Vanjaro/Base.ascx.cs]: It has been added.

**Review by hshimanshusharma — APPROVED** [review]

**Review by hshimanshusharma — APPROVED** [review]

**Review by hshimanshusharma — APPROVED** [review]
